src/Interfaces/LazySet.jl

```diff
@@ -682,3 +683,34 @@ end
 
 isoperationtype(::LazySet) = error("`isoperationtype` cannot be applied to " *
                                    "a set instance; use `isoperation` instead")
+
+"""
+    isequivalent(X::LazySet, Y::LazySet)
+
+Return whether two LazySets are equal in the mathematical sense, i.e. equivalent.
+
+### Input
+
+- `X` -- any `LazySet`
+- `Y` -- another `LazySet`
+
+### Output
+`true` iff `X` is equivalent to `Y`.
+
+### Examples
+
+```
+julia> X = BallInf([0.1, 0.2], 0.3)
+julia> Y = convert(HPolytope, X)
+julia> X == Y
+false
+julia> is_equivalent(X, Y)
+true
+```
+"""
```

Comment:
Let's add the algorithm:

```julia
## Algorithm

First, the check `X == Y` is performed which returns `true` if and only if the given sets are of the same type, and have the same values (modulo floating-point tolerance). Otherwise, the double inclusion check `X ⊆ Y && Y ⊆ X` is computed.
```